### PR TITLE
GPS playback example

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

File: py/examples/file_converter/main.py [hunk 1]
```diff
@@ -104,7 +104,8 @@ def main(
         cv2.waitKey(1)
 
     # close the video writer and the file reader
-    video_writer.release()
+    if video_writer is not None:
```
Comment (Hackerman342) [inline py/examples/file_converter/main.py:107]: Good catch!

File: py/examples/file_reader_gps/main.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    for event_log in gps_events:
+
+        # parse the message
+        msg: gps_pb2.RelativePositionFrame | gps_pb2.GpsFrame = event_log.read_message()
+        gps_msg = unpack_gps_message(msg, msg_type)
+
+        if msg_type == "pvt":
+            print(f"Message stamp: {gps_msg[0]}")
+            print(f"GPS time: {gps_msg[1]}")
+            print(f"Latitude: {gps_msg[2]}")
```
Comment (Hackerman342) [inline py/examples/file_reader_gps/main.py:118]: I think you should remove the function that puts it into a tuple and just print the values directly from the proto. It'd be different if you were creating a class or something, but this just adds risk of indexing error without adding any benefit
```suggestion
            print(f"Message stamp: {msg.stamp.stamp}")
            print(f"GPS time: {msg.gps_time.stamp}")
            print(f"Latitude: {msg.latitude}")
            ...
```

**Review by Hackerman342 — APPROVED** [review]

File: py/examples/file_reader_gps/main.py (hunk 2 above)
Comment (guilhermedemouraa) [reply py/examples/file_reader_gps/main.py]: Done in commit 31429d56be911d2fc592f2de6730cfa3dfd9ad5c

**Review by Hackerman342 — APPROVED** [review]

File: py/examples/file_reader_gps/main.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    gps_events = events_dict[f"/gps/{msg_type}"]
+    print(f"Found {len(gps_events)} packets of gps/{msg_type}\n")
+
+    for event_log in gps_events:
+
+        # parse the message
+        msg: gps_pb2.RelativePositionFrame | gps_pb2.GpsFrame = event_log.read_message()
+
+        if msg_type == "pvt":
```
Comment (edgarriba) [inline py/examples/file_reader_gps/main.py:49]: I’d rather filter by type

Comment (Hackerman342) [reply py/examples/file_reader_gps/main.py:49]: :+1: that also is a good way to show users how to filter by type when they have a stream to all (`'*'`) topics from a service @guilhermedemouraa

**Pull request merged**